### PR TITLE
travis: fix submitting coverage to coveralls. also run cmp.py.

### PR DESCRIPTION
**Pull request by schettino72**
Note: PR cmp-fix must be merged before this.

I added `sudo: false` to use new travis infrastructure. Need to check if it still works..

### CODE REVIEW

**Comment by schettino72** [comment]
one minute. i will rebase this...

File: .travis.yml [hunk 1]
```diff
@@ -5,25 +5,28 @@ python:
...
-  
+
 install:
   - python setup.py install
   - pip install pyflakes
   - pip install pep8
   - pip install docutils
   - pip install nose
-  - pip install coveralls
+  - if [[ $TRAVIS_PYTHON_VERSION == '3.5' && $PYTHONASYNCIODEBUG == '0' ]]; then pip install elasticsearch coverage coveralls; fi
```
Comment (asvetlov) [inline .travis.yml]: if asyncio debug is disabled `PYTHONASYNCIO` is equal to empty string not `0`.
See the latest `travis.yml` version.

Comment (schettino72) [inline .travis.yml]: got it.

**Comment by schettino72** [comment]
@asvetlov looks like some unstable failures... have you seen those before ?
https://travis-ci.org/aio-libs/aioes/builds/84080861

**Comment by schettino72** [comment]
coveralls is working now: https://coveralls.io/github/aio-libs/aioes

**Comment by asvetlov** [comment]
Thanks a lot!

**Pull request merged**